### DISCUSSION TITLE
Dil Seçimi

### DISCUSSION THREAD

**Discussion by eneskasim18** [fatihayaaar/membership-systems#11] [Q&A] [answered]
Neden Java dilini tercih ettin?

**Comment by fatihayaaar** [comment, marked as answer]
Dostum, açıkçası çok bu konu hakkında çük düşündüm. Bunun tek bir nedeni yok tabi ki, başlıca nedenlerimden biri Java dilinde ilerlemiş olmam. Tabi demek değil ki diğer dillere kapalıyım, ileride başka dillerde de bu tarz depolarım olacaktır. :) Java'yı seçme nedenlerimden bir diğeri ise kütüphanelerinin kullanışlılığı ve işimi kolaylaştırması, bir diğer nedende sektörde ki yeri olabilir :)

**Reply by fatihayaaar** [reply]
Dönüşün içinde teşekkür ederim.
